classes.py

```diff
@@ -12,6 +14,10 @@ class Page(TypedDict):
     properties: Dict[str, Property]
     url: str
     
+    @override
```

Comment:
Is `@override` necessary here? Does `Page` have a `__hash__` method in the first place?